src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -54,21 +54,37 @@ trait ExecutionHandle {
   def result: ExecutionResult
 }
 
+sealed trait ReturnCode {
+  def code: Int
+
+  def asScriptReturnCode = this match {
+    case script: ScriptReturnCode => Option(code)
+    case _ => None
+  }
+
+  def asBackendReturnCode = this match {
+    case script: BackendReturnCode => Option(code)
+    case _ => None
+  }
+}
+case class ScriptReturnCode(code: Int) extends ReturnCode
+case class BackendReturnCode(code: Int) extends ReturnCode
```

Comment:
Backend return code feels like it's not a general concept across all backends and might be better in the BACKEND_EXECUTION_INFO table?